ttt-lexical-patterns.lisp

```diff
@@ -9,6 +9,29 @@
 (defparameter *tense* '(past pres cf))
 (defparameter *coordinator* '(and or but because))
 (defparameter *detformer* '(nquan fquan))
+(defparameter *semtypes*
+  '(("([A-Z]|_)*\\.PRO" . "D")
+    ("\\|([A-Z]|_)*\\|" . "D") ; doesn't work properly. Does work with '\|' instead of '|' in the symbol passed to atom-semtype? I think CL doesn't allow | in symbols.
```

Comment:
If you use `util:sym2str` instead of `string` in the `assoc` lookup, this should work.